docs/conceptual/seq.tail['t]-function-[fsharp].md

```diff
@@ -0,0 +1,76 @@
+---
+title: Seq.tail<'T> Function (F#)
+description: Seq.tail<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: liboz
+---
+
+# Seq.tail<'T> Function (F#)
+
+Returns the sequence without the first element.
+
+**Namespace/Module Path:** Microsoft.FSharp.Collections.Seq
+
+**Assembly:** FSharp.Core (in FSharp.Core.dll)
+
+
+## Syntax
+
+```fsharp
+// Signature:
+Seq.tail : seq<'T> -> seq<'T>
+
+// Usage:
+Seq.tail source
+```
+
+#### Parameters
+*source*
+Type: [seq](https://msdn.microsoft.com/library/2f0c87c6-8a0d-4d33-92a6-10d1d037ce75)**&lt;'T&gt;**
+
+
+The input sequence.
+
+## Exceptions
+
+|Exception|Condition|
+|----|----|
+|[ArgumentException](https://msdn.microsoft.com/library/system.argumentexception.aspx)|Thrown when the input sequence is empty.|
+|[ArgumentNullException](https://msdn.microsoft.com/library/system.argumentnullexception.aspx)|Thrown when the input sequence is null.|
+
+## Return Value
+
+The result sequence.
```

Comment:
Let's be specific here.  The resulting sequence is a new sequence composed of the input sequence without its head.  Although that effectively copies the summary, saying "The result sequence" doesn't really add anything.  I'd rather we be slightly redundant.

We could cross-reference `Seq.head` here as well.